code/datums/mutations/hot.dm

```diff
@@ -0,0 +1,31 @@
+/datum/mutation/human/geladikinesis/ash
+	name = "Cindikinesis"
+	desc = "Allows the user to concentrate nearby heat into a pile of ash. Wow. Very interesting."
+	text_gain_indication = "<span class='notice'>Your hand feels warm.</span>"
+	locked = TRUE
+	power_path = /datum/action/cooldown/spell/conjure_item/snow/ash
+
+/datum/action/cooldown/spell/conjure_item/snow/ash
+	name = "Create Ash"
+	desc = "Concentrates pyrokinetic forces to create ash, useful for basically nothing."
```

Comment:
Not quite true. Ash production from nothing is handy for making plastic and mildly for botany to keep plants alive.

Reply:
skill solution